src/sql/workbench/contrib/query/browser/queryResultsEditor.ts

```diff
@@ -122,35 +127,55 @@ export class QueryResultsEditor extends EditorPane {
 	}
 
 	dispose() {
+		if (!this.styleSheet) {
+			return;
+		}
 		this.styleSheet.remove();
 		this.styleSheet = undefined;
 		super.dispose();
 	}
 
 	layout(dimension: DOM.Dimension): void {
+		if (!this.resultsView) {
+			return;
+		}
 		this.resultsView.layout(dimension);
 	}
 
-	setInput(input: QueryResultsInput, options: EditorOptions, context: IEditorOpenContext): Promise<void> {
+	setInput(input: QueryResultsInput, options: EditorOptions, context: IEditorOpenContext): Promise<any> {
```

Comment:
The function you're overriding is a Promise<void> so you shouldn't change that. Instead just don't return anything - which can be done simply by making this function async and deleting the return line